geode-core/src/main/java/org/apache/geode/cache/client/internal/QueueManagerImpl.java

```diff
@@ -109,6 +111,11 @@
   private ScheduledExecutorService recoveryThread;
   private volatile boolean sentClientReady;
 
+  @VisibleForTesting
+  void clearQueueConnections() {
```

Comment:
Since this method is only ever called in a test, it should probably be annotated `@TestOnly`

Reply:
Agreed.

Reply:
Since it is package private it really isn't visible to much other than testing. This annotation is redundant.

Reply:
I will remove it. Thanks!